2021-offseason/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +44,9 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+    new JoystickButton(controller, XboxController.Button.kA.value).whenPressed(new RunCommand(s_intake::startIntake, s_intake)).whenPressed(new RunCommand(s_intake::stopIntake, s_intake));   
+    new JoystickButton(controller, XboxController.Button.kY.value).whenPressed(new RunCommand(s_intake::startInverted, s_intake)).whenPressed(new RunCommand(s_intake::stopIntake, s_intake)); }
```

Comment:
nit: style: place ending bracket on its own line, and maybe space it out. also comments
```suggestion
    // A button -> run intake forward
    new JoystickButton(controller, XboxController.Button.kA.value)
      .whenPressed(new RunCommand(s_intake::startIntake, s_intake))
      .whenPressed(new RunCommand(s_intake::stopIntake, s_intake));
 
    // Y button -> run intake in reverse
    new JoystickButton(controller, XboxController.Button.kY.value)
      .whenPressed(new RunCommand(s_intake::startInverted, s_intake))
      .whenPressed(new RunCommand(s_intake::stopIntake, s_intake));
  }
```

Reply:
it's better now, but @Nakul-Rajpal can you add even more spacing and comments like I suggested here? everything is on the same level so it is hard to tell what method calls are chained to which